android/build.gradle

```diff
@@ -25,6 +25,6 @@ dependencies {
     def supportLibVersion = rootProject.hasProperty('supportLibVersion') ? rootProject.supportLibVersion : DEFAULT_SUPPORT_LIB_VERSION
```

Comment:
I associate the supportLibVersion with "com.android.support". So I would suggest another variable like supportLibVersionAndroidX to distinguish between the two.